app/src/main/java/woowacourse/movie/model/ticket/ReservationTicket.kt

```diff
@@ -0,0 +1,30 @@
+package woowacourse.movie.model.ticket
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import woowacourse.movie.model.movie.ScreeningDateTime
+import woowacourse.movie.model.seats.Seats
+import java.io.Serializable
+
+@Entity(tableName = "reservationTicket")
+data class ReservationTicket(
+    @PrimaryKey(autoGenerate = true)
+    val ticketId: Long = 0,
+    val movieId: Int,
+    val theaterId: Int,
+    val movieTitle: String,
+    val theaterName: String,
+    val seats: Seats,
+    val screeningDateTime: ScreeningDateTime,
+    val amount: Int,
+) : Serializable
+
+fun ReservationTicket.toTicket(): Ticket {
+    return Ticket(
+        movieId = this.movieId,
+        theaterId = this.theaterId,
+        seats = this.seats,
+        screeningDateTime = this.screeningDateTime,
+        amount = this.amount,
+    )
+}
```

Comment:
ReservationTicket 이 하도록 하는건 어떨까요?


```suggestion
@Entity(tableName = "reservationTicket")
data class ReservationTicket(
    @PrimaryKey(autoGenerate = true)
    val ticketId: Long = 0,
    val movieId: Int,
    val theaterId: Int,
    val movieTitle: String,
    val theaterName: String,
    val seats: Seats,
    val screeningDateTime: ScreeningDateTime,
    val amount: Int,
) : Serializable {
    fun toTicket(): Ticket {
        return Ticket(
            movieId = movieId,
            theaterId = theaterId,
            seats = seats,
            screeningDateTime = screeningDateTime,
            amount = amount,
        )
    }
}
```

Reply:
다른 모델에서도 비슷한 경우가 있어서, 함께 수정해보도록 하겠습니다!